jdpostct.c

```diff
@@ -132,6 +132,11 @@ post_process_1pass (j_decompress_ptr cinfo,
   my_post_ptr post = (my_post_ptr) cinfo->post;
   JDIMENSION num_rows, max_rows;
 
+  /* read_and_discard_scanlines may call it with rows "available", but no buffer */
```

Comment:
That seems kinda iffy of `read_and_discard_scanlines`. Is it actually intended to do that?

Reply:
It is iffy, but I don't know that function. I suppose NULL works well for discarding :)

Reply:
Looking at [this](https://github.com/mozilla/mozjpeg/blob/6ee36adc4eaaac0837e86f22ec6c8cafe4b84c92/jdapistd.c#L296-L302), it seems like it's a bit of a hack anyway, so I guess the check is OK. :/